metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/datahub_plugin_v22.py

```diff
@@ -29,6 +29,13 @@
 TASK_ON_RETRY_CALLBACK = "on_retry_callback"
 
 
+def load_config_v22():
+    plugin_config = get_lineage_config()
+    # On the old pluigin we don't support multiple conn_ids, so we just get the first one
+    plugin_config.datahub_conn_id = plugin_config.datahub_conn_id.split()[0].strip()
```

Comment:
we should print a warning here if something is ignored

also, it's not clear to me why the old plugin can't take in a composite hook / composite emitter? or is it mainly just not worth the effort to build?